Equilobe.DailyReport.SL/JiraService.cs

```diff
@@ -26,11 +26,11 @@ public Project GetProject(long id)
             return GetClient(_jiraRequestContext).GetProject(id);
         }
 
-        public List<JiraIssue> GetTimesheetForUser(DateTime startDate, DateTime endDate, string targetUser)
+        public List<JiraIssue> GetTimesheetForUser(string projectKey, string targetUser, DateTime startDate, DateTime endDate)
```

Comment:
Trebuie facuta o clasa context care sa includa toti acesti parametrii. Poti sa faci asta si dupa ce faci merge in master.